galsim/phase_psf.py

```diff
@@ -1068,15 +1080,18 @@ def __str__(self):
 
     def __repr__(self):
         outstr = ("galsim.PhaseScreenPSF(%r, lam=%r, exptime=%r, flux=%r, aper=%r, theta=%r, " +
-                  "scale_unit=%r, interpolant=%r, gsparams=%r)")
+                  "interpolant=%r, scale_unit=%r, gsparams=%r)")
```

Comment:
Ack!  Good catch.  I was attempting to make the repr line up better with the `__init__` arg list, but obviously only half-fixed it.